corehq/apps/smsbillables/models.py

```diff
@@ -312,26 +321,40 @@ def create(cls, message_log, api_response=None):
         country_code, national_number = get_country_code_and_national_number(phone_number)
 
         if backend_instance is None or _sms_backend_is_global(backend_instance):
-            billable.gateway_fee = SmsGatewayFee.get_by_criteria(
-                backend_api_id,
-                direction,
-                backend_instance=backend_instance,
-                country_code=country_code,
-                national_number=national_number,
-            )
-            if billable.gateway_fee is not None:
-                conversion_rate = billable.gateway_fee.currency.rate_to_default
+            if backend_instance is None and backend_api_id == TwilioBackend.get_api_id():
+                def _get_twilio_client():
+                    twilio_backend = MobileBackend.load_by_name(None, "MOBILE_BACKEND_TWILIO")
```

Comment:
it would be better if we load the backend from the backend_id on the sms log entry. i realize that's not entirely possible right now since we currently only store backend_id on outbound sms, but i'm working on making a change so that we always store it on both inbound and outbound sms.

let's plan on there always being a backend_id on the sms and use that to do the lookup. i think my change to set it should be ready for twilio backends by next week sometime.